cmd/zed/agents/zfs_mod.c

```diff
@@ -178,6 +178,7 @@ zfs_unavail_pool(zpool_handle_t *zhp, void *data)
 static void
 zfs_process_add(zpool_handle_t *zhp, nvlist_t *vdev, boolean_t labeled)
 {
+#ifdef HAVE_LIBDEVMAPPER
```

Comment:
auto-online is such a basic feature of zfs that I want to confirm that limiting it to platforms that can build with libdevmapper is acceptable?  What platforms (if any) would be left out?